node-src/ui/messages/info/storybookPublished.ts

```diff
@@ -15,12 +15,12 @@ export default ({ build, storybookUrl }: Pick<Context, 'build' | 'storybookUrl'>
     return dedent(chalk`
       ${success} {bold Storybook published}
       We found ${components} with ${stories}.
-      ${info} View your Storybook at ${link(storybookUrl)}
+      ${info} View your Storybook at ${link(storybookUrl || '')}
```

Comment:
Is this is not present, shouldn't we have an error?  Can we make `storybookUrl` required?

Reply:
> Can we make storybookUrl required?

Yes and we should! I'll fix that up.